cmd/list/rhRegion/cmd.go

```diff
@@ -0,0 +1,71 @@
+package rhRegion
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"text/tabwriter"
+
+	sdk "github.com/openshift-online/ocm-sdk-go"
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/config"
+	"github.com/openshift/rosa/pkg/ocm"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+var (
+	args struct {
+		discoveryURL string
+	}
+)
+
+var Cmd = NewRhRegionCommand()
+
+func NewRhRegionCommand() *cobra.Command {
+	Cmd := &cobra.Command{
+		Use:   "rh-regions",
+		Short: "List available OCM regions",
+		Long:  "List available OCM regions",
```

Comment:
NP for the long can we expand the acronym `OCM` for the most part of ROSA customer OCM is abstracted, so I would like to expand the acronym and even if possible add a doc or flesh out the what/why in the long for this command.

Reply:
Added the acronym for OCM. I'm not sure if we already have the docs available for this. @macgregor can you please share any docs for OCM regionalization, if present?

Reply:
https://docs.google.com/document/d/1WqGRdkeSWd4WEe6S2JubgrA9ESJd0n8RgSrnY9OhxEY/edit#heading=h.bupciudrwmna